orchestrator/cli/database.py

```diff
@@ -0,0 +1,70 @@
+import os
+from shutil import copyfile
+
+import jinja2
+import typer
+from structlog import get_logger
+
+import orchestrator
+
+logger = get_logger(__name__)
+
+app: typer.Typer = typer.Typer()
+
+orchestrator_module_location = os.path.dirname(orchestrator.__file__)
+migration_dir = "migrations"
+
+loader = jinja2.FileSystemLoader(os.path.join(orchestrator_module_location, f"{migration_dir}/templates"))
+jinja_env = jinja2.Environment(
+    loader=loader, autoescape=True, lstrip_blocks=True, trim_blocks=True, undefined=jinja2.StrictUndefined
+)
+
+
+@app.command(name="init")
+def init() -> None:
+    """
+    Run the migrations.
+
+    This command will run the migrations for initialization of the database. If you have extra migrations that need to be run,
+    add this to the
+
+    Returns:
+        None
+
+    """
+
+    if os.access(migration_dir, os.F_OK) and os.listdir(migration_dir):
+        raise OSError(f"Directory {migration_dir} already exists and is not empty")
+
+    logger.info("Creating directory", directory=os.path.abspath(migration_dir))
+    os.makedirs(migration_dir)
+    versions = os.path.join(migration_dir, "versions")
+    logger.info("Creating directory", directory=os.path.abspath(versions))
+    os.makedirs(versions)
+    versions_schema = os.path.join(migration_dir, "versions/schema")
+    logger.info("Creating directory", directory=os.path.abspath(versions_schema))
+    os.makedirs(versions_schema)
+
+    source_env_py = os.path.join(orchestrator_module_location, f"{migration_dir}/templates/env.py.j2")
+    env_py = os.path.join(migration_dir, "env.py")
+    logger.info("Creating file", file=os.path.abspath(env_py))
+    copyfile(source_env_py, env_py)
```

Comment:
Why do we need to copy files? Is that safe?

Reply:
Why cant we do the same thing as we do now? With two folders? except one resides in the python lib?